commit 62bae3fdd6582414f185d92a44e0960bbf668dea
Author: Zack Middleton <zack@cloemail.com>
Date:   Thu Jan 4 05:39:14 2024 -0600

    Update Linux metadata appstream
    
    Flathub requires screenshots and developer_name.

diff --git a/desktop/moe.clover.mm3d.metainfo.xml b/desktop/moe.clover.mm3d.metainfo.xml
index 5afeaa2..d81055e 100644
--- a/desktop/moe.clover.mm3d.metainfo.xml
+++ b/desktop/moe.clover.mm3d.metainfo.xml
@@ -17,9 +17,18 @@
     </p>
   </description>
 
+  <screenshots>
+    <screenshot type="default">
+      <image type="source">https://raw.githubusercontent.com/clover-moe/screenshots/main/mm3d/1.png</image>
+    </screenshot>
+  </screenshots>
+
   <launchable type="desktop-id">moe.clover.mm3d.desktop</launchable>
 
   <url type="homepage">https://clover.moe/mm3d</url>
+  <url type="bugtracker">https://github.com/zturtleman/mm3d/issues</url>
+  <url type="vcs-browser">https://github.com/zturtleman/mm3d</url>
+  <developer_name>zturtleman</developer_name>
   <update_contact>zturtleman@gmail.com</update_contact>
 
   <provides>
